tests/installation/select_patterns_and_packages.pm

```diff
@@ -239,6 +239,9 @@ sub select_specific_patterns_by_iteration {
     # check if we have processed all patterns mentioned in the test suite settings
     my @unseen = keys %patterns;
     die "Not all patterns given in the job settings were processed:" . join(", ", @unseen) if @unseen;
+
+    # accept the patterns and conninue for virtualization tests
+    wait_screen_change { send_key $cmd{acceptlicense} } if (check_var('VIRT_AUTOTEST', '1'));
 }
```

Comment:
this is needed by 12sp1 product only, so such conditions should be there?

Reply:
Not only for sles12sp1, tried on sles12sp3, http://10.67.18.247/tests/748, hung with same problem. So set it for virtualization tests.

Here is the successful one with the fix:
http://10.67.18.247/tests/749

Reply:
This bug has been fixed at #7786  @dzedro , Please review if it is same issue.

Reply:
Should be fixed with #7786